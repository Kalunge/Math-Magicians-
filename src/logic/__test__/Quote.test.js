import React from 'react';
import { expect } from '@jest/globals';
import renderer from 'react-test-renderer';

import Quote from '../../components/Quote';

describe('test quote', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot('pure mathematics is, in its way, the poetry of logical ideas.');
  });
});