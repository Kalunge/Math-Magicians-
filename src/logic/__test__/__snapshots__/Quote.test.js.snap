// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test quote renders correctly: pure mathematics is, in its way, the poetry of logical ideas. 1`] = `
<div
  className="container mb-5"
>
  <div
    className="container mb-5"
  >
    <p
      className="mt-5"
    >
      pure mathematics is, in its way, the poetry of logical ideas.
      <strong>
        — Albert Einstein,German theoretical physicist Without mathematics,
      </strong>
    </p>
    <p
      className="mt-2"
    >
      there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
      <strong>
        — Shakuntala Devi, Indian writer and mental calculator
      </strong>
    </p>
    <p
      className="mt-2"
    >
      Mathematics is the most beautiful and most powerful creation of the human spirit.
      <strong>
        — Stefan Banach, Polish mathematician
      </strong>
    </p>
    <p
      className="mt-2"
    >
      We will always have STEM with us. Some things will drop out of the public eye and go away, but there will always be science, engineering, and technology. And there will always, always be mathematics.
      <strong>
        — Katherine Johnson, African-American mathematician
      </strong>
    </p>
  </div>
</div>
`;
